src/DssSpell.sol

```diff
@@ -53,163 +57,67 @@ contract DssSpellAction is DssAction {
-
-        // ---------- Approve HVBank (RWA009-A) Dao Resolution ----------
-        // Forum: https://forum.makerdao.com/t/huntingdon-valley-bank-transaction-documents-on-permaweb/16264/24
-
-        // Approve HVBank (RWA009-A) Dao Resolution with IPFS hash QmStrc9kMCmgzh2EVunjJkPsJLhsVRYyrNFBXBbJAJMrrf
+        // ---------- Increase the GSM Pause Delay ----------
+        // Forum: https://forum.makerdao.com/t/gsm-pause-delay-increase-proposal/23929
+        // Poll: https://vote.makerdao.com/polling/QmcLsYwj
+
+        // Increase the GSM Pause Delay by 14 hours, from 16 hours to 30 hours -----
+        PauseLike(MCD_PAUSE).setDelay(30 hours);
+
+        // ---------- Add the following ilks to LINE_MOM ----------
+        // Forum: https://forum.makerdao.com/t/gov12-1-2-bootstrapping-edit-proposal-gov10-2-3-1a-edit/24005
+        // Poll: https://vote.makerdao.com/polling/QmZsAM36
+
+        // ETH-A
+        LineMomLike(LINE_MOM).addIlk("ETH-A");
+        // ETH-B
+        LineMomLike(LINE_MOM).addIlk("ETH-B");
+        // ETH-C
+        LineMomLike(LINE_MOM).addIlk("ETH-C");
+        // WSTETH-A
+        LineMomLike(LINE_MOM).addIlk("WSTETH-A");
+        // WSTETH-B
+        LineMomLike(LINE_MOM).addIlk("WSTETH-B");
+        // WBTC-A
+        LineMomLike(LINE_MOM).addIlk("WBTC-A");
+        // WBTC-B
+        LineMomLike(LINE_MOM).addIlk("WBTC-B");
+        // WBTC-C
+        LineMomLike(LINE_MOM).addIlk("WBTC-C");
+
+        // ---------- Spark MetaMorpho Vault Parameters ----------
+        // Forum: https://forum.makerdao.com/t/morpho-spark-dai-vault-update-1-april-2024/24006
+        // Forum: https://forum.makerdao.com/t/morpho-spark-dai-vault-update-1-april-2024/24006/8
+
+        // DDM DC-IAM Parameters: line: 1 billion DAI
+        DssExecLib.setIlkAutoLineDebtCeiling("DIRECT-SPARK-MORPHO-DAI", 1 * BILLION);
+
+        // ---------- Approve TACO Resolution ----------
+        // Forum: https://forum.makerdao.com/t/bt-project-ethena-risk-legal-assessment/23978
+
+        // Approve IPFS Resolutions: Qmf8Nv4HnTFNDwRgcLzRgBdtVsVVfKY2FppaBimLK9XhxB
         // Note: see `dao_resolutions` variable declared above
 
         // ---------- Spark Proxy Spell ----------
-        // Forum: https://forum.makerdao.com/t/mar-6-2024-proposed-changes-to-sparklend-for-upcoming-spell/23791/
-        // Forum: https://forum.makerdao.com/t/stability-scope-parameter-changes-11-under-sta-article-3-3/23910
-        // Poll: https://vote.makerdao.com/polling/QmQAtb17
-        // Poll: https://vote.makerdao.com/polling/QmYYoAMe
+        // Forum: https://forum.makerdao.com/t/mar-21-2024-proposed-changes-to-sparklend-for-upcoming-spell/23918
+        // Poll: https://vote.makerdao.com/polling/QmdjqTvL
+        // Poll: https://vote.makerdao.com/polling/QmaEqEav
+        // Poll: https://vote.makerdao.com/polling/QmbCWUAP
 
         // Trigger Spark Proxy Spell at 0x210DF2e1764Eb5491d41A62E296Ea39Ab56F9B6d
```

Comment:
Right, the address is wrong, but I'd still keep the word "Trigger" instead of "Pass", because that is the established pattern. I asked Governance to update the sheet.

Reply:
Thanks! The exec sheet is updated so we can update instruction here as well :)